Formula/clap-host.rb

```diff
@@ -0,0 +1,24 @@
+class ClapHost < Formula
+  desc "CLAP example host"
+  homepage "https://github.com/free-audio/clap-host"
+  url "https://github.com/free-audio/clap-host", using: :git, tag: "1.0", revision: "b5db1c246c173692371a17d92c2bd5fda1826d98"
+  license "MIT"
+  head "https://github.com/free-audio/clap-host", using: :git, branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "qt"
+  depends_on "rtaudio"
+  depends_on "rtmidi"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", "-DCLAP_BUILD_TESTS=TRUE", "-DCLAP_HOST_BUNDLE=FALSE", *std_cmake_args
+    system "cmake", "--build", "build", "--target", "install"
```

Comment:
```suggestion
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```
Let's follow the pattern used by other formulae.